app/controllers/admin/bulk_republishing_controller.rb

```diff
@@ -30,13 +30,58 @@ def republish
     end
   end
 
+  def new_documents_by_organisation; end
+
+  def search_documents_by_organisation
+    @organisation = Organisation.find_by(slug: params[:organisation_slug])
+
+    unless @organisation
+      flash[:alert] = "Organisation with slug '#{params[:organisation_slug]}' not found"
+      return redirect_to(admin_bulk_republishing_documents_by_organisation_new_path)
+    end
+
+    redirect_to(admin_bulk_republishing_documents_by_organisation_confirm_path(params[:organisation_slug]))
+  end
+
+  def confirm_documents_by_organisation
+    unless @organisation&.slug == params[:organisation_slug]
```

Comment:
I think I'm misunderstanding here, but when would this evaluate to `true`? Would `@organisation` be defined here?

Reply:
We did this in the non-bulk controller. Initially (when my Rails was a bit more rusty) I wasn't sure if the instance variable might be retained from earlier actions, but everything should be dropped between requests. I wasn't sure here whether, because we're using a redirect to get to this action, there's a chance the instance variable will be retained. But I'm pretty sure the redirect will just lead to an entirely new request, so we should probably delete this conditional logic here and elsewhere!

Reply:
Oops, I failed to remove this